process-engine/bpmn-studio
# ✨Add cronjob events

## Changes

1. Use events to update cronjobs

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/390
Part of #1594 

PR: #1739 

## How to test the changes

- open the dashboard view where you can see the cronjob list
- now the cronjob list should get updated when you create, update, stop, execute or remove a cronjob/diagram with a cycle timer start event
